pkg/lvgl/Makefile.dep

```diff
@@ -0,0 +1,18 @@
+USEMODULE += lvgl_core
+USEMODULE += lvgl_draw
+USEMODULE += lvgl_font
+USEMODULE += lvgl_hal
+USEMODULE += lvgl_misc
+USEMODULE += lvgl_objx
+USEMODULE += lvgl_themes
+
+USEMODULE += lvgl_contrib
```

Comment:
I would strongly prefer to have this optional. In case a user wants to handle the LVGL thread himself.

Reply:
Ok, done!